pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1476,4 +1476,22 @@ self: super: {
 
   persistent-mysql = dontCheck super.persistent-mysql;
 
+  EdisonAPI = appendPatch super.EdisonAPI (pkgs.fetchpatch {
```

Comment:
Could you add a comment to both `EdisonAPI` and `EdisonCore` explaining what the patch is for, and when you expect it can be removed?

Reply:
Sure, added.